test/elements.js

```diff
@@ -49,7 +49,7 @@ test('svg', function (t) {
   </svg>`
   t.equal(result.tagName, 'svg', 'create svg tag')
   t.equal(result.childNodes[0].tagName, 'rect', 'created child rect tag')
-  t.equal(result.childNodes[1].getAttribute('xlink:href'), '#test', 'created child use tag with xlink:href')
+  t.equal(result.childNodes[2].getAttribute('xlink:href'), '#test', 'created child use tag with xlink:href')
```

Comment:
think we're close, but so far the goal has been to remove artifacts like these - e.g. when we're creating lists of items that we must reorder, not having any random text nodes in between is a requirement. I should've expressed that as a test tho, my bad - but this test should be changed back